app/assets/stylesheets/pages/_review.scss

```diff
@@ -0,0 +1,67 @@
+h3{
+  margin-left:100px;
+  margin-top:50px;
+  margin-bottom:30px;
+  font-weight: bold;
+  }
+
+  h5{
+  margin-left:100px;
+  }
+
+.rating {
+  unicode-bidi: bidi-override;
+  direction: rtl;
+}
+.rating > span {
+  display: inline-block;
+  position: relative;
+  width: 1.1em;
+}
+.rating > span:hover:before,
+.rating > span:hover ~ span:before {
+   content: "\2605";
+   position: absolute;
+}
+
+.review-container {
+  display: flex;
+  width: 70%;
+  border: 2px solid #F5F5F5;
+  padding: 50px;
+  margin-left: 100px;
+  margin-top:40px;
+
+  .first-flex-item div {
```

Comment:
try a more specific css class selector. what does this flex item actually contain ?